eunkyung/services/userService.js

```diff
@@ -0,0 +1,37 @@
+const userDao = require("../models/userDao");
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+
+const createUser = async (name, password, email, profileImage) => {
+  const saltRounds = 12;
+  const hashedPw = await bcrypt.hash(password, saltRounds);
+
+  const createUserResult = await userDao.createUser(
+    name,
+    hashedPw,
+    email,
+    profileImage
+  );
+  return createUserResult;
+};
+
+const emailCheckLogin = async (email, password) => {
```

Comment:
함수명은 login 또는 signin으로 작성해도 괜찮을 것 같습니다.
```suggestion
const login = async (email, password) => {
```